components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/SequenceConfig.java

```diff
@@ -46,6 +49,7 @@
     private String authenticatedIdPs;
 
     private AuthenticatorConfig authenticatedReqPathAuthenticator;
+    private List<String> requestedAcr;
```

Comment:
If "requestedAcr" a run time data, why is it a part of this class which is a configuration data?